src/FourteenDayActiveUsersMetric.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Tightenco\NovaGoogleAnalytics;
+
+use Illuminate\Support\Arr;
+use Illuminate\Http\Request;
+use Laravel\Nova\Metrics\Trend;
+use Laravel\Nova\Metrics\TrendResult;
+use Spatie\Analytics\Analytics;
+use Spatie\Analytics\Period;
+use Carbon\Carbon;
+
+class FourteenDayActiveUsersMetric extends Trend
+{
+    use ActiveUsersTrait;
+
+    public function name()
+    {
+        return __('14 Day Active Users');
+    }
+
+    /**
+     * Calculate the value of the metric.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return mixed
+     */
+    public function calculate(Request $request)
+    {
+        $lookups = [
+            5 => $this->performQuery('ga:14dayUsers', 5),
```

Comment:
This is probably clear to someone with a higher view of the app, but how does this work? Is this "Users who have been active within the 14 days prior to [5 days ago, 10 days ago, 15 days ago]"?

If so, do we also need to add one for 0, or is that already handled?

Reply:
Yes, it's users who have been active for 14 days compared with 5, 10, or 15 days. No need to add one for 0 as 0 is not an available option. Defaults to 5 days.